examples/todo/data/db.json

```diff
@@ -1,13 +1,18 @@
 {
   "ids": {
-    "Todo": 5
+    "Todo": 10
   },
   "models": {
     "Todo": {
       "1": "{\"title\":\"Take over the galaxy\",\"desc\":\"MWAHAHAHAHAHAHAHAHAHAHAHAHAMWAHAHAHAHAHAHAHAHAHAHAHAHA\",\"id\":1}",
       "2": "{\"title\":\"destroy alderaan\",\"desc\":\"Make sure there are no survivors left!\",\"id\":2}",
       "3": "{\"title\":\"play space invaders\",\"desc\":\"Become the very best!\",\"id\":3}",
-      "4": "{\"title\":\"crush rebel scum\",\"desc\":\"Every.Last.One.\",\"id\":4}"
+      "4": "{\"title\":\"crush rebel scum\",\"desc\":\"Every.Last.One.\",\"id\":4}",
+      "5": "{\"title\":\"a todo sample\",\"desc\":\"Something to do.\",\"id\":5}",
+      "6": "{\"title\":\"a todo sample\",\"desc\":\"Something to do.\",\"id\":6}",
+      "7": "{\"title\":\"a todo sample\",\"desc\":\"Something to do.\",\"id\":7}",
+      "8": "{\"title\":\"a todo sample\",\"desc\":\"Something to do.\",\"id\":8}",
+      "9": "{\"title\":\"a todo sample\",\"desc\":\"Something to do.\",\"id\":9}"
```

Comment:
Please don't forget to revert changes in the sample data too when removing sample observers.

Reply:
🙈 sure, need to clean up them every time run tests on local